CHANGELOG.md

```diff
@@ -1,5 +1,62 @@
 # CHANGELOG
 
+## 1.29.0 (2022-09-16)
```

Comment:
Can we make sure that all of the features / bug fixes with the same tag (e.g. `[Tracking]`) are located next to each other (i.e. don't interleave them) and that Pipelines is first, followed by Tracking, UI, Models, Model Registry, Scoring?